microsoft/testsuites/ltp/ltp.py

```diff
@@ -340,14 +344,29 @@ def _install(self) -> bool:
         self.node.tools[Mkdir].create_directory(top_src_dir, sudo=True)
         self.node.tools[Chmod].update_folder(top_src_dir, "a+rwX", sudo=True)
 
-        # clone ltp
-        git = self.node.tools[Git]
-        ltp_path = git.clone(
-            self.LTP_GIT_URL, cwd=PurePosixPath(top_src_dir), dir_name=top_src_dir
-        )
+        if self._tar_file_path:
+            self._log.debug(f"Use downloaded ltp tar: {self._tar_file_path}!")
+            self._remote_tar_path = self.get_tool_path(
+                use_global=True
+            ) / os.path.basename(self._tar_file_path)
+
+            mkdir = self.node.tools[Mkdir]
+            mkdir.create_directory(self._remote_tar_path.parent.as_posix())
+            self.node.shell.copy(PurePath(self._tar_file_path), self._remote_tar_path)
+            self.node.tools[Tar].extract(
+                str(self._remote_tar_path), top_src_dir, sudo=True
```

Comment:
don't extract it to the src dir. The src dir is to find a free partition for build, but for the prebuilt package, the system disk should be enough. Use the global tool path.

Reply:
This should be good now as it's source code.